src/databases/Uintah/avtUintahFileFormat.C

```diff
@@ -114,7 +110,7 @@ avtUintahFileFormat::avtUintahFileFormat(const char *filename,
       dataVariesOverTime = attrs->GetBool(UINTAH_DATA_VARIES_OVER_TIME);
     }
   }
-
+  
```

Comment:
It looks like you added a space to this blank line.

Reply:
Accident when forcing a recompile. Will send a fix??? How best to do that ? a new commit??

Reply:
I see that change (second commit) in now.